apps/antalmanac/src/stores/AppStore.ts

```diff
@@ -299,6 +341,29 @@ class AppStore extends EventEmitter {
 
     termsInSchedule = (term: string) =>
         new Set([term, ...this.schedule.getCurrentCourses().map((course) => course.term)]);
+
+    autoSaveSchedule = (action: any[] = [null, null, null]) => {
+        const autoSave = typeof Storage !== 'undefined' && window.localStorage.getItem('autoSave') == 'true';
+        if (autoSave) {
+            const savedUserID = window.localStorage.getItem('userID');
+
+            if (savedUserID != null) {
+                saveSchedule(savedUserID, true, true).then((r) => void r);
```

Comment:
What is this void for?

```suggestion
                saveSchedule(savedUserID, true, true);
```